### DISCUSSION TITLE
Links For You

### DISCUSSION THREAD

**Discussion by giscus** [cfjedimaster/raymondcamden2023#10] [Comments]
# Links For You

Lucious links for your enjoyment.

https://www.raymondcamden.com/2024/01/13/links-for-you

**Comment by bobmonsour** [comment]
Thank you for the shout out, Ray! Thankfully, I continue enjoy maintaining said "bundle."

**Comment by cfjedimaster** [comment]
For folks noticing Bob's comment on every "Links for You" post - this was my fault. When I implemented Giscus, I told it to use the title of the post as the unique identifier, which is usually safe, but I've used "Links For You" as a common title for this series. I don't believe I can fix it as I would lose existing comments (unless maybe I manually moved them which I don't want to do), so going forward, I will title my L4U posts with the date in them as well. That's a bit unnecessary as my posts have dates displayed, but feels like the best fix for this.

**Reply by bobmonsour** [reply]
I hadn't noticed.

**Reply by cfjedimaster** [reply]
I saw it today on my new L4U post and was like... wtaf!?!?
